src/Joomlatools/Console/Command/Vhost/Alias.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * @copyright	Copyright (C) 2007 - 2019 Johan Janssens and Timble CVBA. (http://www.timble.net)
+ * @license		Mozilla Public License, version 2.0
+ * @link		http://github.com/joomlatools/joomlatools-console for the canonical source repository
+ */
+
+namespace Joomlatools\Console\Command\Vhost;
+
+use Joomlatools\Console\Joomla\Util;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+
+use Joomlatools\Console\Command\Site\AbstractSite;
+
+class Alias extends AbstractSite
+{
+    protected function configure()
+    {
+        parent::configure();
+
+        $this
+            ->setName('vhost:alias')
+            ->setDescription('Manage aliases for virtual hosts')
+            ->addArgument(
+                'alias',
+                InputArgument::REQUIRED,
+                'Virtual host alias'
+            )
+            ->addOption(
+                'delete',
+                'D',
+                InputOption::VALUE_NONE,
+                'Delete the alias if it exists.'
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        parent::execute($input, $output);
+
+        if (!file_exists($this->target_dir)) {
+            throw new \RuntimeException(sprintf('Site not found: %s', $this->site));
+        }
+
+        $site   = $this->site;
+        $alias  = $input->getArgument('alias');
+        $delete = $input->getOption('delete');
+
+        $changed = $this->_updateAliases($site, $alias, $delete, 'apache');
+        $changed = $this->_updateAliases($site, $alias, $delete, 'nginx') || $changed;
+
+        if ($changed && Util::isJoomlatoolsBox())
+        {
+            `box server:restart apache nginx`;
```

Comment:
@stevenrombauts just following this terminal command:
https://github.com/joomlatools/joomlatools-vagrant/blob/master/puppet/environments/box/modules/profiles/files/box/cli/scripts/Command/Server/Restart.php#L50

It thought we were supposed to be using `sudo systemctl restart apache2` from now on?